server/server_test.go

```diff
@@ -1322,12 +1324,28 @@ func startTestServer(config string) error {
 	if err != nil {
 		return err
 	}
+
 	go func() {
 		err := server.Start()
 		if err != nil {
 			return
 		}
 	}()
+
+	retry := 3
```

Comment:
You should look at this the other way: this constant is hidden somewhere in more than 1300 lines source file. A common and good practice is to expose configurable constants at the beginning of a source file - that's why golang has const section. If there are many common constants, it may also be a good idea to have a separate source file - but this is not the case here.